### PR TITLE
Just some really small changes

### PR DESCRIPTION
**Pull request by distractedm1nd**
Hey! Love the contract. Feel free to delete this PR, just some small things I personally find better

### CODE REVIEW

File: contracts/SmartHoldETH.sol [hunk 1]
```diff
@@ -6,7 +6,7 @@ import "./PriceFeedInterface.sol";
 
 contract SmartHoldETH {
   address public owner = msg.sender;
-  uint public depositedAt = block.timestamp;
+  uint public depositedAt = now;
```
Comment (pawurb) [inline contracts/SmartHoldETH.sol]: `now` is deprecated in favour of `block.timestamp` in future versions of solidity

**Review by pawurb — CHANGES_REQUESTED** [review]

**Comment by pawurb** [comment]
Thanks for the PR. Please apply fixes and squash commits into 1 and I'm looking forward to merging it.

File: contracts/SmartHoldETH.sol (hunk 1 above)
Comment (distractedm1nd) [reply contracts/SmartHoldETH.sol]: Ohhh. Glad to hear that, I didn't know. Will do!

**Comment by pawurb** [comment]
Thanks!

**Pull request merged**